docs/Documentation/Systems/Vermillion/running.md

```diff
@@ -335,12 +335,203 @@ task    thread             node name  first task    # on node  core
 0001      0001                 c1-32        0000         0001  0099
 ```
 
+# Running VASP on Vermilion
 
-## Running VASP on CPUs
+A few different versions of VASP are available on Vermilion:
+- [VASP 5 (Intel MPI)](#Running-VASP-5-with-IntelMPI-on-CPUs)
+- [VASP 6 (Intel MPI)](#Running-VASP-6-with-IntelMPI-on-CPUs)
+- [VASP 6 (Open MPI)](#Running-VASP-6-with-OpenMPI-on-CPUs)
+- [VASP 6 on GPUs](#Running-VASP-6-on-GPUs)
 
-The batch script given above can be modified to run VASP. VASP with Open MPI is recommended.
+Running VASP with Open MPI shows a small improvement compared to running with Intel MPI, and running VASP on GPUs shows an even larger improvement.
 
-To load a build of VASP that is compatible with Open MPI:
+VASP runs faster on 1 node than on 2 nodes. In some cases, VASP run times on 2 nodes have been observed to be double (or more) the run times on a single node. Many issues have been reported for running VASP on multiple nodes. In order for MPI to work successfully on Vermilion, it is necessary to specify the interconnect network that Vermilion should use to communicate between nodes. This is documented in each of the scripts below. The documented recommendations for setting the interconnect network have been shown to work well for multi-node jobs on 2 nodes, but aren't guaranteed to produce succesful multi-node runs on 4 nodes. 
+
+If many cores are needed for your VASP calcualtion, it is recommended to run VASP on a singe node in the lg partition (60 cores/node), which provides the largest numbers of cores per node. 
+
+## Running VASP 5 with IntelMPI on CPUs
+
+To load a build of VASP 5 that is compatible with Intel MPI (and other necessary modules):
+
+```
+module use  /nopt/nrel/apps/220525b/level01/modules/lmod/linux-rocky8-x86_64/gcc/12.1.0/
+ml vasp/5.5.4
+ml intel-oneapi-mkl
+ml intel-oneapi-compilers
+ml intel-oneapi-mpi
+```
+
+This will give you:
+
+```
+[myuser@vs example]$ which vasp_gam
+/nopt/nrel/apps/220525b/level01/install/opt/spack/linux-rocky8-zen2/gcc-12.1.0/vasp544/bin/vasp_gam
+[myuser@vs example]$ which vasp_ncl
+/nopt/nrel/apps/220525b/level01/install/opt/spack/linux-rocky8-zen2/gcc-12.1.0/vasp544/bin/vasp_ncl
+[myuser@vs example]$ which vasp_std
+/nopt/nrel/apps/220525b/level01/install/opt/spack/linux-rocky8-zen2/gcc-12.1.0/vasp544/bin/vasp_std
+```
+
+Note the directory might be different. 
+
+In order to run on more than one node, we need to specify the network interconnect. To do so, use mpirun instead of srun. We want to use "en7" as the interconnect. The mpirun command looks like this. 
+
+```
+I_MPI_OFI_PROVIDER=tcp mpirun -iface ens7 -np 16 vasp_std
```

Comment:
iface names don't match. Which one is correct?

Reply:
Should be "ens7". Thanks for the catch

Reply:
Fixed it in running.md